File: backend/src/main/java/com/team05/petmeeting/domain/animal/dto/external/AnimalBody.java
package com.team05.petmeeting.domain.animal.dto.external;

import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
public class AnimalBody {
    private AnimalItems items;
    private Integer numOfRows;
    private Integer pageNo;
    private Integer totalCount;
}

File: backend/src/main/java/com/team05/petmeeting/domain/animal/dto/external/AnimalBody.kt
package com.team05.petmeeting.domain.animal.dto.external

data class AnimalBody @JvmOverloads constructor(
    var items: AnimalItems? = null,
    var numOfRows: Int? = null,
    var pageNo: Int? = null,
    var totalCount: Int? = null,
)
